sale_triple_discount/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 QubiQ (http://www.qubiq.es)
+# Copyright 2015 ADHOC SA  (http://www.adhoc.com.ar)
+# Copyright 2017 Alex Comba - Agile Business Group
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Triple Discount',
+    'version': '11.0.1.0.0',
+    'category': 'Sales Management',
+    'author': 'QubiQ, '
+              'ADHOC SA, '
+              'Agile Business Group, '
+              'Tecnativa, '
+              'Odoo Community Association (OCA)',
```

Comment:
Authorname should be added when you added a feature my friend. If not, add your name to contributors in readme.